.pre-commit-config.yaml

```diff
@@ -46,13 +46,15 @@ repos:
     rev: v1.11.2
     hooks:
     -   id: mypy
-        exclude: dbt-adapters/src/dbt/adapters/events/adapter_types_pb2.py|dbt-tests-adapter/src/dbt/__init__.py
```

Comment:
don't we want to exclude _pbt2.py?

Reply:
Again, I thought this would be an issue, but it's somehow no longer an issue. I was concerned it wasn't running at all, but it wouldn't have generated all of the conflicts that I found if it weren't running. And it found issues in the `dbt-adapters` package.